server/bingo/models.py

```diff
@@ -1,3 +1,77 @@
-from django.db import models # noqa
+from django.db import models
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
 
-# Create your models here.
+
+class UserManager(BaseUserManager):
+
+    def create_user(self, username, email, password=None):
+
+        user = self.model(
+            username=username,
+            email=self.normalize_email(email)
+        )
+
+        user.set_password(password)
+        user.save(using=self.db)
+        return user
+
+    def create_superuser(self, username, email, password=None):
+        user = self.create_user(
+            username=username,
+            email=email,
+            password=password
+        )
+        user.is_admin = True
+        user.save(using=self.db)
+        return user
+
+
+class User(AbstractBaseUser):
+    REQUIRED_FIELDS = ["email"]
+
+    user_id = models.BigAutoField(primary_key=True)
```

Comment:
I think a ``BigAutoField`` is unnecessary here. You can change it to an ``AutoField``.